tfjs-backend-webgpu/src/kernels/conv2d_mm_vec4_webgpu.ts

```diff
@@ -107,6 +146,13 @@ export class Conv2DMMVec4Program implements WebGPUProgram {
         void mm_write(int row, int col, vec4 value) {
           if (row < dimAOuter && col < dimBOuter)
           {
+            ivec4 outCoord = ivec4(
+              batch,
+              row / ${this.outputShape[2]},
+              row % ${this.outputShape[2]},
+              col * 4);
+            ${addBiasSnippet}
+            ${applyActivationSnippet}
             result[batch * ${batchSize} + row * dimBOuter + col] = value;
```

Comment:
Maybe it's better to change `result[batch * ${batchSize} + row * dimBOuter + col] = value` to `setOutput(outCoord, value)`.